src/activities/Elsa.Activities.AzureServiceBus/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -70,5 +70,27 @@ private static ManagementClient CreateServiceBusManagementClient(IServiceProvide
             var connectionString = options.ConnectionString;
             return new ManagementClient(connectionString);
         }
+
+        private static IServiceBusQueuesStarter CreateServiceBusQueuesStarter(IServiceProvider serviceProvider)
+        {
+            var options = serviceProvider.GetRequiredService<IOptions<AzureServiceBusOptions>>().Value;
+            var multitenancyEnabled = options.MultitenancyEnabled;
+
+            if (multitenancyEnabled)
+                return ActivatorUtilities.GetServiceOrCreateInstance<MultitenantServiceBusQueuesStarter>(serviceProvider);
+            else
+                return ActivatorUtilities.GetServiceOrCreateInstance<ServiceBusQueuesStarter>(serviceProvider);
+        }
+
+        private static IServiceBusTopicsStarter CreateServiceBusTopicsStarter(IServiceProvider serviceProvider)
+        {
+            var options = serviceProvider.GetRequiredService<IOptions<AzureServiceBusOptions>>().Value;
+            var multitenancyEnabled = options.MultitenancyEnabled;
+
+            if (multitenancyEnabled)
```

Comment:
Is there any reason *not* to always assume multi-tenancy (with DEFAULT being the default tenant)?

Reply:
Good point, I've had a reason not to do it earlier, but then I changed implementation and now it makes sense to get rid of this check and additional 'multitenant' classes.